### PR TITLE
Draft - Add CAIP-2 for NEAR Protocol

### PR DESCRIPTION
**Pull request by ALuhning**
Draft of CAIP-2 for NEAR Protocol.

### CODE REVIEW

File: CAIPs/caip-draft_near_protocol.md
Comment (ligi) [inline CAIPs/caip-draft_near_protocol.md:65]: I would add one `}` to have both blocks closed

File: CAIPs/caip-draft_near_protocol.md
Comment (ligi) [inline CAIPs/caip-draft_near_protocol.md:10]:
```suggestion
requires: 2
```

File: CAIPs/caip-draft_near_protocol.md
Comment (ligi) [inline CAIPs/caip-draft_near_protocol.md:15]:
```suggestion
Defines the NEAR Protocol namespace and reference for CAIP-2.
```

File: CAIPs/caip-draft_near_protocol.md
Comment (ligi) [inline CAIPs/caip-draft_near_protocol.md:33]:
```suggestion
The definition for this namespace will use the `chain_id` as an identifier for each of the three NEAR Protocol chains. The format is a string equal to either `mainnet`, `testnet` or `betanet`.  The chain identifier is appended by a [chain id](https://chainid.network).
```

File: CAIPs/caip-draft_near_protocol.md
Comment (ligi) [inline CAIPs/caip-draft_near_protocol.md:62]: not a big fan of this tbh - IMHO "testnet" is a bad ID - especially as in this context a chain_id is usually an EIP155 number

**Review by ligi — CHANGES_REQUESTED** [review]
Some nits and one discussion item

File: CAIPs/caip-draft_near_protocol.md
Comment (TrevorJTClarke) [reply CAIPs/caip-draft_near_protocol.md:62]: Good point -- This is part of the root shard response. We will bring up with the NEAR Core team and see if we can define a better standards payload item/id.

Comment (oed) [reply CAIPs/caip-draft_near_protocol.md:62]: Any updates here? Would be good to have this merged :)

Comment (pedrouid) [reply CAIPs/caip-draft_near_protocol.md:62]: I think it would be a good idea to merge this as is right now.

The CAIP spec includes a Resolution Method for this namespace which is exactly what is required.

The feedback is related to core logic on Near clients which will reflect any changes through the Resolution Method described so this shouldn't need to block the PR.

**Review by jekyleee — APPROVED** [review]

**Comment by pedrouid** [comment]
Another one that looks good to merge @ligi

**Review by pedrouid — APPROVED** [review]
👍

File: CAIPs/caip-draft_near_protocol.md
Comment (tahpot) [reply CAIPs/caip-draft_near_protocol.md:62]: @ligi Can this be merged? I agree this shouldn't be held up by core logic in NEAR clients.

**Comment by tahpot** [comment]
Can this be merged? We want to use it at Verida.

**Comment by oed** [comment]
All blockchain specific specifications are moving to the namespaces repo: https://github.com/ChainAgnostic/namespaces/pulls
Please submit a PR there for this to be added!

**Comment by obstropolos** [comment]
Closing this here - as mentioned previously, please make all namespace PRs in the namespaces repo:
https://github.com/ChainAgnostic/namespaces/pulls

**Pull request closed without merging**
